### DISCUSSION TITLE
Unified FuzzySearch API

### DISCUSSION THREAD

**Discussion by taooceros** [Flow-Launcher/Flow.Launcher#1229] [Ideas]
I think we can add a field in the Result class to include an `IEnumerable<string>` property that allows a unified fuzzy match API matching. Therefore, plugins won't need to do the fuzzy search themselves.

**Comment by jjw24** [comment]
By plugins you are referring to non-C/F# plugins?

I think that's a good idea but I'm still thinking how we could achieve it though, can you explain a bit more?

**Reply by taooceros** [reply]
> By plugins you are referring to non-C/F# plugins?
> 
> I think that's a good idea but I'm still thinking how we could achieve it though, can you explain a bit more?

Both should be able to use (C#/Non C/F#)

**Comment by raindropsfromsky** [comment]
What is the syntax for using the fuzzy search?

**Reply by taooceros** [reply]
Not sure for now. Maybe plugins are just required to pass a string for matching?
